### PR TITLE
Change: Use gender-neutral pronouns

### PR DESCRIPTION
**Pull request by unbeatable-101**
## Motivation / Problem

Same as #9189, OpenTTD shouldn't be using gendered pronouns in English.

## Description

This PR changes the single usage of "his/her" in `console_cmds.cpp` to "their".

Further reasoning is the same as in #9189.

## Limitations

I can't see this causing any issues

## Checklist for review

Some things are not automated, and forgotten often. This list is a reminder for the reviewers.
* The bug fix is important enough to be backported? (label: 'backport requested')
* This PR affects the save game format? (label 'savegame upgrade')
* This PR affects the GS/AI API? (label 'needs review: Script API')
    * ai_changelog.hpp, gs_changelog.hpp need updating.
    * The compatibility wrappers (compat_*.nut) need updating.
* This PR affects the NewGRF API? (label 'needs review: NewGRF')
    * newgrf_debug_data.h may need updating.
    * [PR must be added to API tracker](https://wiki.openttd.org/en/Development/NewGRF/Specification%20Status)

### CODE REVIEW

**Comment by frosch123** [comment]
Maybe also fix the comments. Some of them end up in the AI/GS/source documentation.
I grepped about 25 cases, though many of the "he" are either mistyped "the" or they should really be "it".

**Comment by frosch123** [comment]
There are also some occurences of "boy", even in console_cmds.cpp.

File: src/network/network_client.cpp [hunk 1]
```diff
@@ -194,7 +194,7 @@ NetworkRecvStatus ClientNetworkGameSocketHandler::CloseConnection(NetworkRecvSta
  */
 void ClientNetworkGameSocketHandler::ClientError(NetworkRecvStatus res)
 {
-	/* First, send a CLIENT_ERROR to the server, so he knows we are
+	/* First, send a CLIENT_ERROR to the server, so it knows we are
 	 *  disconnection (and why!) */
```
Comment (FLHerne) [inline src/network/network_client.cpp]: might as well fix "we are disconnection"

File: src/network/network_client.cpp [hunk 2]
```diff
@@ -1399,7 +1399,7 @@ void NetworkClientSetCompanyPassword(const char *password)
 }
 
 /**
- * Tell whether the client has team members where he/she can chat to.
+ * Tell whether the client has team members where they can chat to.
```
Comment (FLHerne) [inline src/network/network_client.cpp]: Drop "where" here?

File: src/network/network_server.cpp [hunk 3]
```diff
@@ -1855,14 +1855,14 @@ void NetworkServer_Tick(bool send_frame)
 
 			case NetworkClientSocket::STATUS_MAP_WAIT:
 				/* Send every two seconds a packet to the client, to make sure
-				 * he knows the server is still there; just someone else is
+				 * It knows the server is still there; just someone else is
```
Comment (FLHerne) [inline src/network/network_server.cpp]: Wrong case, this continues from the line above.

**Review by unbeatable-101 — COMMENTED** [review]
"Silly boy, you cannot move the server!"

I guess just "You cannot move the server!" should suffice?

Edit: I thought this would be inline like the comments above were, it's line 809 of console_cmds.cpp

**Comment by FLHerne** [comment]
> "Silly boy, you cannot move the server!"
> 
> I guess just "You cannot move the server!" should suffice?

I'd think so.

"Fool! You cannot..." would preserve the meaning if there's a need to gratuitously insult the player at this point. There isn't.

File: src/network/network_client.cpp (hunk 2 above)
Comment (unbeatable-101) [reply src/network/network_client.cpp]: I think replace it with "who"

**Comment by FLHerne** [comment]
For the future, please consider finishing the immediate set of changes before pushing rather than many small updates in quick succession.

Every time you update the PR, it sends an email to everyone who's interacted with it and a message to the IRC channel. This is getting a bit spammy...

Thanks.

File: src/network/network_client.cpp (hunk 2 above)
Comment (FLHerne) [reply src/network/network_client.cpp]: Yes, that's better.

**Comment by unbeatable-101** [comment]
> For the future, please consider finishing the immediate set of changes before pushing rather than many small updates in quick succession.
> 
> Every time you update the PR, it sends an email to everyone who's interacted with it and a message to the IRC channel. This is getting a bit spammy...
> 
> Thanks.

Sorry, I was using github instead of downloading it and pushing them from my device since I just expected to be doing the one change.

**Comment by 2TallTyler** [comment]
Somewhat unrelated and not necessarily a question for OP, but why are console messages literal strings and not references to translatable strings?

**Comment by unbeatable-101** [comment]
> Somewhat unrelated and not necessarily a question for OP, but why are console messages literal strings and not references to translatable strings?

I guess that is something else to work on, I opened a suggestion [here](https://github.com/OpenTTD/OpenTTD/discussions/9250)

**Comment by daisyjreyn** [comment]
any update?

**Comment by LordAro** [comment]
An update on what, exactly?

**Pull request merged**